android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -64,6 +64,8 @@
 import com.google.android.exoplayer2.upstream.DefaultBandwidthMeter;
 import com.google.android.exoplayer2.util.MimeTypes;
 import com.google.android.exoplayer2.util.Util;
+//Import PlayerControlView
```

Comment:
Please remove these comments that simply state what the codes does. Comments are valuable when they describe why you're doing something or cover assumptions you're making about how things work. However, when the code is obvious it's simply redundant.

You can keep the documentation headers on the functions.

Reply:
Here's a good guide for writing quality comments:
https://medium.freecodecamp.org/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83